exe/todo

```diff
@@ -3,14 +3,13 @@
 Encoding.default_external = Encoding::UTF_8
 Encoding.default_internal = Encoding::UTF_8
 
-lib_path = File.expand_path("../../lib", __FILE__)
-$LOAD_PATH.unshift(lib_path) unless $LOAD_PATH.include?(lib_path)
-
-deps = %w(cli-ui cli-kit)
-deps.each do |dep|
-  vendor_path = File.expand_path("../../vendor/deps/#{dep}/lib", __FILE__)
-  $LOAD_PATH.unshift(vendor_path) unless $LOAD_PATH.include?(vendor_path)
-end
+unshift_path = ->(path) {
+  p = File.expand_path("../../#{path}", __FILE__)
+  $LOAD_PATH.unshift(p) unless $LOAD_PATH.include?(p)
+}
+unshift_path.call('vendor/deps/cli-ui/lib')
+unshift_path.call('vendor/deps/cli-kit/lib')
+unshift_path.call('lib')
 
 require 'todo'
 # CLI::Kit::ReportErrors.setup(logfile, Todo::ErrorReporter)
```

Comment:
I think we can uncomment this now?

Reply:
I should actually make it work. The API here is not very clean. That would work if we had an `ErrorReporter`